string task_name
string object_name
string action
float32 frequency
---
int32 error
string topic_name